unittests/AST/ASTImporterTest.cpp

```diff
@@ -4012,6 +4024,26 @@ TEST_P(ASTImporterLookupTableTest,
   EXPECT_EQ(Res.size(), 0u);
 }
 
+TEST_P(ASTImporterLookupTableTest,
+       LookupFindsFriendClassDeclWithTypeAlias) {
+  TranslationUnitDecl *ToTU = getToTuDecl(
+      R"(
+      class F;
+      using alias_of_f = F;
+      class Y { friend alias_of_f; };
+      )",
+      Lang_CXX11);
+
+  ASTImporterLookupTable LT(*ToTU);
+
+  auto *FriendD = FirstDeclMatcher<FriendDecl>().match(ToTU, friendDecl());
+  auto *F = FirstDeclMatcher<CXXRecordDecl>().match(ToTU, cxxRecordDecl(hasName("F")));
+
+  const RecordDecl *RD = getRecordDeclOfFriend(FriendD);
+
+  EXPECT_EQ(F, RD);
```

Comment:
I still do not understand what this test should do. The `LT` variable is not used at all, but it is the lookup table that should be tested. In the current form this test only creates a TU and does searches on the AST of that TU but the lookup table is not used for anything.

Reply:
Perhaps we should have a comment above the ctor call that we test there is no crash in the ctor.

Reply:
The comment should contain that the constructor should not crash, and why the next checks are made (if only the constructor is tested these would be unneeded).

Reply:
Actually, it is possible to have a test without an `EXPECT` call, that is okay.